index.d.ts

```diff
@@ -1,5 +1,5 @@
 import { FastifyPluginCallback, FastifySchema, onRequestHookHandler, preHandlerHookHandler } from 'fastify';
-import { OpenAPI, OpenAPIV2, OpenAPIV3 } from 'openapi-types';
+import { OpenAPI, OpenAPIV2, OpenAPIV3, OpenAPIV3_1 } from 'openapi-types';
```

Comment:
If we use OpenAPIV3_1 then we dont need OpenAPIV3. So modify in line the namespace merging by changing `namespace OpenAPIV3` to `namespace OpenAPIV3_1`

Reply:
Okay, I updated it.